quantum/via.c

```diff
@@ -329,12 +318,19 @@ void raw_hid_receive(uint8_t *data, uint8_t length) {
 #endif
             break;
         }
+#ifdef VIA_EEPROM_ALLOW_RESET
+        case id_eeprom_reset: {
+            via_eeprom_set_valid(false);
+            eeconfig_init_via();
+            break;
+        }
+#endif
         case id_dynamic_keymap_macro_get_count: {
             command_data[0] = dynamic_keymap_macro_get_count();
             break;
         }
         case id_dynamic_keymap_macro_get_buffer_size: {
-            uint16_t size   = dynamic_keymap_macro_get_buffer_size();
+            uint16_t size   = dynamic_keymap_macroi_get_buffer_size();
```

Comment:
```suggestion
            uint16_t size   = dynamic_keymap_macro_get_buffer_size();
```
There's no I in ~~team~~ macro

Reply:
It can't be one of my PRs unless there is a spelling error :D